.gitignore

```diff
@@ -1 +1,4 @@
 **/.package-lock.json
+/.nyc-output
```

Comment:
Not sure what's causing this but when I run `npm run test` on my end the folder being created is `.nyc_output` (with an underscore instead of a hyphen).

For completeness, the version of nyc I have installed locally is:

```shell
$ npm ls nyc      
licensee@9.0.0 .../licensee.js
└─┬ tap@16.3.1
  └── nyc@15.1.0
```

and [the documentation for that version](https://www.npmjs.com/package/nyc/v/15.1.0) does indeed list `"./.nyc_output`" as the default temp-dir. I don't know if tap is supposed/used to override this though.

Reply:
I don't have it in front of me right now but there's a decent chance that this was a typo on my part.